keras_cv/backend/ops.py

```diff
@@ -14,9 +14,22 @@
 from keras_cv.backend.config import multi_backend
 
 if multi_backend():
+    import keras_core
     from keras_core.backend import vectorized_map  # noqa: F403, F401
     from keras_core.operations import *  # noqa: F403, F401
+    from keras_core.operations import arange as keras_core_arange  # noqa
+
+    def arange(start, stop=None, step=1, dtype=None):
```

Comment:
```suggestion
    # Note that we can directly call the arange op in tensorflow
    # because keras core doesn't treat any inputs as symbolic
    # tensors and, hence, doesn't put the op on the graph.
    def arange(start, stop=None, step=1, dtype=None):
```